tests/python/gaia-ui-tests/gaiatest/tests/accessibility/gallery/manifest.ini

```diff
@@ -0,0 +1,11 @@
+[parent:../manifest.ini]
+
+[DEFAULT]
+external = false
+sdcard = true
+
+[test_gallery_frame_visibility.py]
+smoketest = true
+# Bug 1103106 - Assertion failure: !mWillChangeBudgetCalculated
+skip-if = device == "desktop" && os == "linux" && debug
```

Comment:
I copied this from:
https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/manifest.ini#L19

Since the test is very similar, I kept that skip condition.